_docs-sources/intro/overview/products.md

```diff
@@ -0,0 +1,31 @@
+---
+'title': 'Products'
+---
+
+# Gruntwork Products
+
+## What we provide
```

Comment:
Is this just an outline, or the proposed format? The bulleted list format feels a bit lacking.